src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -83,9 +100,18 @@ public CommandManagerWindow(DriverCommandManager commandManager) {
 		content.add(btnClearObservers, c);
 	}
 
+	public void updateCurrentCommandPreviewPanel() {
+		DriverCommand currentCommand = commandManager.getCurrentCommand();
+		previewPanelDrawerController.clearPanel();
+		if (currentCommand != null) {
+			currentCommand.execute(driver);
+		}
+	}
```

Comment:
Try to extract this function to separate class and remove new dependencies. This feature requires only access to one panel in GUI, it does not need to be a part of CommandManagerWindow.